frontend/src/components/pages/Contact.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from 'react'
-import { useNavigate } from 'react-router-dom';
+import { Navigate, useNavigate } from 'react-router-dom';
```

Comment:
Remove unused import
```suggestion
import { useNavigate } from 'react-router-dom';
```